spec/integration/review_spec.rb

```diff
@@ -1,14 +1,31 @@
 require 'spec_helper'
 
-describe 'project show page with reviews' do
-  it 'shows all reviews in a project' do
-    project = create(:project, title: "Java Tic-Tac-Toe")
-    review = create(:review, content: "Looks good!", project_id: project.id)
+describe 'review' do
+  describe 'project show page with reviews' do
+    it 'shows all reviews in a project' do
+      project = create(:project, title: "Java Tic-Tac-Toe")
+      review = create(:review, content: "Looks good!", project_id: project.id)
 
-    visit '/projects/' + project.id.to_s
+      visit '/projects/' + project.id.to_s
 
-    page.has_content? project.title
-    page.has_content? project.description
-    page.has_content? review.content
+      page.has_content? project.title
+      page.has_content? project.description
+      page.has_content? review.content
+    end
+  end
+
+  describe 'new review' do
+    it 'creates a new review' do
+      project = create(:project, title: "Java Tic-Tac-Toe")
+      content = 'This looks really good!'
+      new_review_page = '/projects/' + project.id.to_s + '/reviews/new'
+
+      visit new_review_page
+      fill_in new_review_page[content], with: content, visible: false
+      click_button('Submit')
+      visit '/projects/' + project.id.to_s
+
+      page.has_content? content
```

Comment:
I think that this needs to be the RSpec syntax:

expect(page).to have_content(content)